src/Rule.php

```diff
@@ -2271,6 +2302,18 @@ public function showMinimalCriteriaForm($fields, $canedit, $rand)
      * @param $addtotd   (default '')
      **/
     public function getMinimalCriteriaText($fields, $addtotd = '')
+    {
+        $to_display = $this->getMinimalCriteria($fields);
+        $text  = "<td $addtotd>" . $to_display['criterion'] . "</td>";
+        $text .= "<td $addtotd>" . $to_display['condition'] . "</td>";
+        $text .= "<td $addtotd>" . $to_display['pattern'] . "</td>";
+        return $text;
+    }
+
+    /**
+     * @param $fields
+     **/
+    public function getMinimalCriteria($fields)
```

Comment:
1. This method seems to be used only locally, it should be declared as private.
2. Types hinting should be added.

```suggestion
    private function getMinimalCriteria(array $fields): array
```